client/my-sites/google-my-business/select-business-type/index.js

```diff
@@ -140,8 +176,10 @@ class GoogleMyBusinessSelectBusinessType extends Component {
 				mainText={ translate(
 					"Don't provide in-person services? Learn more about reaching your customers online."
 				) }
-				buttonText={ translate( 'Optimize Your SEO', { comment: 'Call to Action button' } ) }
-				buttonHref={ `/settings/traffic/${ siteSlug }` }
+				buttonTarget="_blank"
+				buttonText={ translate( 'Learn More about SEO', { comment: 'Call to Action button' } ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Learn more about Search.' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
👎